Packs/ThreatZone/Integrations/ThreatZone/ThreatZone.yml

```diff
@@ -144,13 +141,26 @@ script:
         - name: entry_id
           description: Entry ID of the file to submit.
           required: true
+        - name: private
+          required: false
+          auto: PREDEFINED
+          predefined:
+            - "true"
+            - "false"
+          description: If this is true then everyone can see the submisison. If this is false then only your workspace participants can see the submission
```

Comment:
Maybe I'm missing something, but it seems to me that it should be the other way around, if `private=true` then doesn't everyone have to see?